dll/win32/rpcrt4/rpcrt4.spec

```diff
@@ -378,7 +378,7 @@
 378 stdcall RpcErrorStartEnumeration(ptr)
 379 stub RpcFreeAuthorizationContext
 380 stdcall RpcGetAsyncCallStatus(ptr) RpcAsyncGetCallStatus
-# RpcGetAuthorizationContextForClient
+381 stdcall RpcGetAuthorizationContextForClient(ptr long ptr ptr long long ptr ptr ptr)
```

Comment:
The **last 3** parameters should read: `long ptr ptr)` (since "Reserved3" is DWORD) ?

Reply:
And the LUID should be an `int64`

Reply:
Fixed. Thanks.